test/mux.sh

```diff
@@ -0,0 +1,353 @@
+      <match journal>\n\
+        @type rewrite_tag_filter\n\
+        rewriterule1 MESSAGE .+ project.testproj.mux\n\
+        rewriterule2 message .+ project.testproj.mux\n\
+      </match>\n\
+      <filter project.testproj.mux>\n\
+        @type record_transformer\n\
+        enable_ruby\n\
+        <record>\n\
+        @timestamp ${time.strftime(\"%Y-%m-%dT%H:%M:%S%z\")}\n\
+        </record>\n\
+      </filter>"}]'
+  elif [ $myoption -eq $SET_CONTAINER_VALS ]; then
+      oc patch configmap/logging-fluentd --type=json --patch '[{ "op": "replace", "path": "/data/filter-pre-mux-test-client.conf", "value": "\
+      <match journal>\n\
+        @type rewrite_tag_filter\n\
+        rewriterule1 MESSAGE .+ project.testproj.mux\n\
+        rewriterule2 message .+ project.testproj.mux\n\
+      </match>\n\
+      <filter project.testproj.mux>\n\
+        @type record_transformer\n\
+        enable_ruby\n\
+        <record>\n\
+        @timestamp ${time.strftime(\"%Y-%m-%dT%H:%M:%S%z\")}\n\
+        CONTAINER_NAME k8s_mux.01234567_logging-mux_testproj_00000000-1111-2222-3333-444444444444_55555555\n\
+        CONTAINER_ID_FULL 0123456789012345678901234567890123456789012345678901234567890123\n\
+        </record>\n\
+      </filter>"}]'
+  elif [ $myoption -eq $MISMATCH_NAMESPACE_TAG ]; then
+      oc patch configmap/logging-fluentd --type=json --patch '[{ "op": "replace", "path": "/data/filter-pre-mux-test-client.conf", "value": "\
+      <match journal>\n\
+        @type rewrite_tag_filter\n\
+        rewriterule1 MESSAGE .+ project.bogus.mux\n\
+        rewriterule2 message .+ project.bogus.mux\n\
+      </match>\n\
+      <filter project.bogus.mux>\n\
+        @type record_transformer\n\
+        enable_ruby\n\
+        <record>\n\
+        @timestamp ${time.strftime(\"%Y-%m-%dT%H:%M:%S%z\")}\n\
+        CONTAINER_NAME k8s_mux.01234567_logging-mux_testproj_00000000-1111-2222-3333-444444444444_55555555\n\
+        CONTAINER_ID_FULL 0123456789012345678901234567890123456789012345678901234567890123\n\
+        </record>\n\
+      </filter>"}]'
+  elif [ $myoption -eq $NO_PROJECT_TAG ]; then
+      oc patch configmap/logging-fluentd --type=json --patch '[{ "op": "replace", "path": "/data/filter-pre-mux-test-client.conf", "value": "\
+      <match journal>\n\
+        @type rewrite_tag_filter\n\
+        rewriterule1 MESSAGE .+ test.bogus.mux\n\
+        rewriterule2 message .+ test.bogus.mux\n\
+      </match>\n\
+      <filter test.bogus.mux>\n\
+        @type record_transformer\n\
+        enable_ruby\n\
+        <record>\n\
+        @timestamp ${time.strftime(\"%Y-%m-%dT%H:%M:%S%z\")}\n\
+        </record>\n\
+      </filter>"}]'
+  else
+      os::log::info "Enabling secure forward"
+  fi
+
+  reset_fluentd_daemonset
+
+  os::log::debug "$( oc label node --all logging-infra-fluentd=true )"
+  os::cmd::try_until_text "oc get pods -l component=fluentd" "^logging-fluentd-.* Running "
```

Comment:
Hm right -- I had to do this: https://github.com/openshift/origin-aggregated-logging/blob/master/test/cluster/rollout.sh#L47-L48

Might be more robust than this